### PR TITLE
Block LMS player loots from loot tracker

### PR DESCRIPTION
**Pull request by psikoi**
Prevents any loot from player kills inside Last Man Standing from appearing in the loot tracker

Also fixed a typo in the herbiboar bit

### CODE REVIEW

**Comment by deathbeam** [comment]
It should be always saved, like hidden drops always are, and just be hidden from UI.

Also, instead of config option, imo simply skipping player drop events inside of LMS is better, because noone cares about that there (and that was also basically the request of the reporter)

**Comment by psikoi** [comment]
@deathbeam updated, as requested

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java [hunk 1]
```diff
@@ -115,6 +116,9 @@
 		13113, "Larran's small chest"
 	);
 
+	// Last man standing map regions
+	private static final int[] LAST_MAN_STANDING_REGIONS = {13658, 13659, 13914, 13915, 13916};
```
Comment (ypperlig) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java]: Should this be an immutable set? Reference:
https://github.com/runelite/runelite/pull/1286#discussion_r179928302

Comment (psikoi) [reply runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java]: Good call, should be good now

**Comment by deathbeam** [comment]
needs rebase

**Comment by Adam-** [comment]
I don't see why the items need to be saved at all. They aren't even real.

**Comment by deathbeam** [comment]
Yeah, for LMS specifically my previous comment about saving is not relevant. The original change was about hiding player loot (by not saving it).

**Pull request merged**